src/sfdt/__tests__/fixtures/nestedConditionListWithOneChildToggleOff.js

```diff
@@ -0,0 +1,70 @@
+export const inlines = [
+  {
+      "bookmarkType": 0,
+      "name": "COND::dafe554d-08b5-463f-a40c-cf5e260be606"
```

Comment:
this fixture is pretty short, perhaps we can generated using our helper functions.